tests/replication/test_pusher_shard.py

```diff
@@ -114,6 +118,10 @@ def test_send_push_single_worker(self):
             ],
         )
 
+    # Pusher functionality is active on the main process by default. Disable that by
+    # explicitly overriding what 'pusher_instances' will be available so
+    # ShardedWorkerHandlingConfig will be filled out correctly.
+    @override_config({"pusher_instances": ["pusher1", "pusher2"]})
```

Comment:
If we're providing this are shared config, can we remove it from the `extra_config`?

Reply:
Yep